packages/wonder-blocks-dropdown/src/components/multi-select.tsx

```diff
@@ -217,74 +190,77 @@ type State = Readonly<{
  * </MultiSelect>
  * ```
  */
-export default class MultiSelect extends React.Component<Props, State> {
-    labels: Labels;
-
-    static defaultProps: DefaultProps = {
-        alignment: "left",
-        disabled: false,
-        error: false,
-        light: false,
-        shortcuts: false,
-        selectedValues: [],
-    };
-
-    constructor(props: Props) {
-        super(props);
-
-        this.state = {
-            open: false,
-            searchText: "",
-            lastSelectedValues: [],
-            // merge custom labels with the default ones
-            labels: {...defaultLabels, ...props.labels},
-        };
-        // merge custom labels with the default ones
-        this.labels = {...defaultLabels, ...props.labels};
```

Comment:
`labels` on the class wasn't being used so it can be removed. We combine default labels and prop labels:

https://github.com/Khan/wonder-blocks/blob/e02ada1e89fd30c08b34fabba5658e2547541640/packages/wonder-blocks-dropdown/src/components/multi-select.tsx#L220-L221